rightscale/resource_network.go

```diff
@@ -28,7 +38,7 @@ func resourceNetwork() *schema.Resource {
 				ForceNew:    true,
 			},
 			"deployment_href": &schema.Schema{
-				Description: "ID of deployment that owns network",
+				Description: "ID of deployment that is associated with the network",
```

Comment:
owns the seems better to me?

Reply:
Correct me if I'm wrong, but only if you use it that way, no?  Networks are top level objects much like deployments, and if you want to associate them by scoping the network to a deployment then you can do this, but it's not required.

N deployments can contain servers, instances, server arrays that use the same network.

Reply:
*if* you use a deployment it is a container - you don't have to.

Reply:
Also I guess it should say `href` and not `ID` so maybe something like:
```
Optional href of deployment that owns the network
```

Reply:
AH - ok so if I provide a href of a deployment when I create a network then the network truly is scoped ONLY to that deployment?  I did not know that!  :100: